pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -575,14 +575,14 @@ func (b *volumeBinder) bindAPIUpdate(ctx context.Context, pod *v1.Pod, bindings
 	for _, binding = range bindings {
 		klog.V(5).InfoS("bindAPIUpdate: binding PV to PVC", "pod", klog.KObj(pod), "PV", klog.KObj(binding.pv), "PVC", klog.KObj(binding.pvc))
 		// TODO: does it hurt if we make an api call and nothing needs to be updated?
-		klog.V(2).InfoS("Claim bound to volume", "PVC", klog.KObj(binding.pvc), "PV", klog.KObj(binding.pv))
+		klog.V(2).InfoS("Updating PersistentVolume: binding to claim", "PVC", klog.KObj(binding.pvc), "PV", klog.KObj(binding.pv))
```

Comment:
I think the only difference between this and the above log message is the pod. I think it's fine to combine into one.

Reply:
I combined into one and tried to keep consistency.